core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -15,3 +15,105 @@
+  var subscriptionDashboardPage = null;
+
+  var creator1Id = 'FirstCreator';
+  var creator2Id = 'SecondCreator';
+  users.createUser('FirstUser@profileFeatures.com',
+    'FirstUserprofileFeatures');
+  users.createUser('SecondUser@profileFeatures.com',
+    'SecondUserprofileFeatures');
+  users.createUser(creator1Id + '@profileFeatures.com',
+    creator1Id);
+  users.createUser(creator2Id + '@profileFeatures.com',
+    creator2Id);
+
+  beforeAll(function() {
+    libraryPage = new LibraryPage.LibraryPage();
+    learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
+    subscriptionDashboardPage =
+      new SubscriptionDashboardPage.SubscriptionDashboardPage();
+  });
+
+  it('View a particular exploration created by another User', function() {
+    users.login(creator1Id + '@profileFeatures.com');
+    workflow.createAndPublishExploration(
+      'Activation',
+      'Chemistry',
+      'English'
+    );
+    users.logout();
+
+    users.login('FirstUser@profileFeatures.com');
+    // Subscribing to a User to get all his created explorations.
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator1Id);
+    subscriptionDashboardPage.navigateToSubscriptionButton();
+
+    // finding a particular exploration on user's library and playing it.
+    libraryPage.get();
+    libraryPage.findExploration('Activations');
+    libraryPage.playExploration('Activations');
+    users.logout();
+  });
+
+  it('Subscribe to a particular User', function() {
+    users.login('FirstUser@profileFeatures.com');
+
+    // Subscribe to both the creators.
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator1Id);
+    subscriptionDashboardPage.navigateToSubscriptionButton();
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator2Id);
+    subscriptionDashboardPage.navigateToSubscriptionButton();
+
+    // Checking all the users subscribed present in user's dashboard.
+    // Both creators should be present in the subscriptions section of the
+    // dashboard.
+    learnerDashboardPage.get();
+    learnerDashboardPage.navigateToSubscriptionsSection();
+    // The last user (SecondCreator) that learner subsribes to is placed first
+    // in the list.
+    learnerDashboardPage.expectSubscriptionFirstNameToMatch('Second...');
+    // The first user (FirstCreator) that learner subscribes to is placed
+    // last in the list.
+    learnerDashboardPage.expectSubscriptionLastNameToMatch('First...');
+    users.logout();
+  });
+
+  it('View number of explorations created by a User', function() {
+    users.login('FirstUser@profileFeatures.com');
+
+    // Creating some explorations.
+    workflow.createAndPublishExploration(
+      'Activation',
+      'Chemistry',
+      'Introduction',
+      'English',
+      'BUS101',
+      'Business'
+    );
+
+    // Getting Number of explorations created by a User.
+    LibraryPage.get();
+    LibraryPage.getAllExplorationElements();
```

Comment:
Ummm, what are you doing with this function call? I don't see any follow up assertions.

Reply:
`getAllExplorationElements()` was created to get the number of total explorations created.

Reply:
But after getting the number, you should check it somewhere. You cannot have tests that just get some values. There should be an assertion. (Like here make sure the value returned is 6)

Reply:
@nithusha21 I went through the list of functions available in the `LibraryPage.js`, but no function is available to check if the value returned is 6 or not. So in that case, should i create a function?

Reply:
For this, I've created another function in `LibraryPage.js` to check the number of explorations created.